docs/lifecycle/lfn_entry/opx.md

```diff
@@ -1,2 +1,282 @@
+* Issues list: (Login to github required) [https://github.com/issues?q=is%3Aopen+is%3Aissue+user%3Aopen-switch+sort%3Aupdated-desc](https://github.com/issues?q=is%3Aopen+is%3Aissue+user%3Aopen-switch+sort%3Aupdated-desc) 
+* OPX uses the Github issue tracker for all of its bugs.
+#### Chat tooling
+* Project RocketChat
+    * [https://chat.openswitch.net](https://chat.openswitch.net)
+    * OPX community quite active in chat channels. 
+    * Links to chat tooling used by the project.
+    * Overview of chat tooling used by the candidate project.
+    * Users have the option of creating private channels with each other if necessary.
+#### Code repositories
+* [https://github.com/open-switch](https://github.com/open-switch) 
+    * 59 public repositories 
+    * Provides a list of all OPX repositories which are mapped to architecture components
+        * [https://github.com/open-switch/opx-docs/wiki/Repositories](https://github.com/open-switch/opx-docs/wiki/Repositories)   
+#### Code review
+    * OPX uses the Github Pull Request interface for code review
+    * Overview of code review norms, practices, conventions, rules.
+    * To what extent are external/private code review systems used? None
+#### Continuous Integration tooling
+* CodeFactor
+* BuildKite, whose pipeline consists of:
+    * Setup
+        * Docker Hub
+    * Build
+        * dbp
+    * Smoketest
+        * Docker Hub
+        * AWS
+    * Github DCO
+#### Documentation
+* [https://github.com/open-switch/opx-docs/wiki](https://github.com/open-switch/opx-docs/wiki) 
+
+#### Mailing lists
+* [https://lists.openswitch.net/](https://lists.openswitch.net/)
+* dev@ primary mailing list 556 subscribers. 
+* Public TSC list, Private Governing board list.
+* Active chat forum [https://chat.openswitch.net](https://chat.openswitch.net)  with 285 subscribers 
+* To what extent are external/private mailing lists used? None
+
+#### Meeting calendars
+* Weekly public TSC meetings.
+    * [https://github.com/open-switch/opx-docs/wiki/Technical-Steering-Committee-Meeting-Details](https://github.com/open-switch/opx-docs/wiki/Technical-Steering-Committee-Meeting-Details) * Weekly meeting minutes (PPT files) archived in tsc mailing list.  [https://lists.openswitch.net/g/tsc/topics](https://lists.openswitch.net/g/tsc/topics) 
+    * To what extent are meetings public, and clearly publicly documented?
+* Meeting minutes
+    * Weekly meeting presentations are archived in tsc mailing list.  [https://lists.openswitch.net/g/tsc/topics](https://lists.openswitch.net/g/tsc/topics) 
+    * TSC call recordings are available if requested.  
+    * To what extent are public minutes for meetings taken and shared? Generally the meeting topics are documented in the TSC powerpoint files. Open action items are tracked in the last slide(s) of the PPT files and are followed up at the next week's meeting.
+
+### Integrations
```

Comment:
OPNFV is a project under LFN which focuses on integrating different open source communities together and run e2e testing to verify that certain use cases work. It is possible to integrate the different projects under LFN but I wonder how we could do that with OPX. Do we need special hardware? Is it possible to run OPX in a regular x86 server even if not all functionality is available? Anything you could comment in this regard would be great as it could trigger some future collaboration between OPX and OPNFV

Reply:
As it is today, OPX does offer ability to run in virtualized environment with limited functionality on the NPU side. We already do a lot of testing with cluster of VM's using GNS3. The current VM is supported under x86 server. Here is more information about OPX VM and how to run it: https://github.com/open-switch/opx-docs/wiki/Hardware-virtualization 
We also provide vagrant boxes for all releases of OPX. Here is a link to the latest version of OPX: https://app.vagrantup.com/opx/boxes/3.1.0
We look forward to work with OPNFV and other LFN projects to better align OPX and provide necessary support to integrate/cooperate.
Of course, we could always provide hardware, if virtualized limitations block our progress.